.github/pull_request_template.md

```diff
@@ -1,7 +1,11 @@
-[Optional] 
-Fixes #issue
-
-*<Context of what this change does, why it is needed and how it is accomplished>*
+Describe context of what this change does,
```

Comment:
ubernit: Personally, I kind of prefer 1 sentence per line for Mardown so it works better for diffing. Also, the Github UI handles text wrapping very well when writing PRs. I'm not against this change, just wanted to point out my personal preference.